### PR TITLE
App updated for submission

### PR DESCRIPTION
**Pull request by Ayush-kumar1**
Guys pls review my component library

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,699 @@
...
+    left: -34px;
+}
+
+.button_head {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+#btn-default {
```
Comment (sneha-pujari) [inline style.css:250]: `cursor: pointer` could be added, since it is good practice for a better UI.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,676 @@
...
+        <!-- SIDEBAR  -->
+
+        <div id="page-wrap">
+            <div class="blok" id="sidebar">
+
+                <!-- <br><br> -->
+
+                <h3>Components</h3>
+
+                <ul>
```
Comment (sneha-pujari) [inline index.html:51]: 'Introduction - Usage / installation instructions' could be added to the the sidenav for better navigation.

**Review by sneha-pujari — COMMENTED** [review]
Hello, 
I have commented on the code. 
The rest of the code looks good.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Component Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
```
Comment (jugalkpatel) [inline index.html:11]: add a favicon here.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,676 @@
...
+            </div>
+        </div>
+
+
+
+
+        <!-- SIDEBAR  -->
+
+        <div id="page-wrap">
```
Comment (jugalkpatel) [inline index.html:44]: sidebar is overlapping with main content in responsive mode.
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,676 @@
...
+            <div class="blok" id="sidebar">
```
Comment (jugalkpatel) [inline index.html:45]: add padding on the left of the sidebar.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,676 @@
...
...vtE47Ge4O/V0Dny7SBHiLGdmd2LOzFmZt5ZjxJl99jOzO7FnYlmZjmzN0kyhhIsfeK1tDRMpWWDOTidfgPAccdlYjxRRpqdSnoauVbeMwydOIp7gyk+gTz5qeTfBeFxyvl7WhHcn5RAVR/ce6TJzFK1UgZC9x0Aohax4HwtW0B7arNusqfGHqPmmTRIRZk9LaN8E+aj3Nju+SfgzOZVpjb0zdh9j7bvLNUFRYOjPly915ipLFSWKksCSksT5cqgkmEloSSCTt1DL1VsRgmRzG49I3HvnQEj7E9dUb6evHXjhYfPk3rlw0veOdT2r+Uo2I2xGGzztwe7eeaGWRnFo3BdX+OXfBq7m/OL/Hbfg1/3fnOQpIlJPrlR3zwXuii0tC6zp634Ffc35yJ1ht+BX3N+c4ykrZJ3rtR3zw5KQhh7oXYdYrvgVdzfnIHWa/4FXc/5ziZJS6TvXajvlFbBB3Au5taMR8Cn6Nn6pS+t2K5ko+jZ+qcD1yiyudFZP3yjsp6fuBddutmM5jWvZX+ZM83FawY2zjiLFHQmzX6VAMayuctiQgnldi4701HDA3Bg3BcFpLEsxJJ4sxLMe0nfKmWdViyhhGIyng61c7CUuJ0uJQ4lhEVJUNFJNK4+1dUoWOTzRBw+FFjjKzEEOcxkRWPEU95P+qYzUnVw4y7bsH7vUwLk8HbiKx9/V2wvATzzoWeyzVizoGnxd7D33J5XdUrqVYZg8RLIoIi0WFZ1ZzG6Nao5jNk6ecdv5zmVZrJw36ORt49qer8pTVGS896LcfY89+p6OszWP3rxAskFnc+jrBwybsP5yo4dxxU+mV7qeRuLTuR+lacCqdmNsy7wZ6D3R9mQuL15U7MbZl+zG2Z66vXlSVkCs6CsiVnrq7eXMVkGWdJWRZZG6iBy5GWVMk62WVMk9YitcuN1nNYk73Sc7pPBMNcuCxJy2pPRsSctiQrUyxy821ZyWLPStrPQZzvh3PBGPYpMbjTLXhec4lTz1l0Li28XD3n+k4HeRPQweo+MtPtwlS9NjDa8yjP7pZRAnQuvqYYxa9wHn7LIvNHqvqjdjCHs2qsPxLkZM46EB4+Vw7ZtdD6kYWgh7M77B8MKKwekJ+ZM1IGUea4gKnq8tCwtg/8AR9hz3LnwODrorWupQqKMgo9Z6T1zpiinFniSTaV//9k=" class="image3">
```
Comment (jugalkpatel) [inline index.html:176]: try to use svgs in your imgs.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,676 @@
...
...7g3HMSARzisRjktWLrBHlfuujCVbcajbXQ61QfJTY8AjwsDHIf3vUZVPd3J+5MDqgdWJ9iKeeyMzFCQhOikyVHrzFTcZ2axFqwcSyTau2SojVlPxIXEaBgYHyMEgUmabHRaRP8A0fkXAiMwXKxQk/dJzj6x7V09eDggFW9DrXG+xloM7r3kXWCNatnRbgUkXfF98BgQOgPtvBxq5ai1ZaQNjvrzieXQUyuWY48CbFiRsrGCddJG2/n/AHFS1tQd53rFYdsXcMvcMa7cto0/varPD2GQGHLjTQnoeQiAfTeKbhi2aDPHOfb+tRLmIuC4qqFylTmORpERBzZojU6QdqQnWGnTXU+39iouNVTcQspbUbE+azl22Zp9Km0iZy39Irk4+63RbRUD/hDefWsbxJiUbwn22iN4Ht860X6R7+bil9dxltE8jHdW5HlpWaxV1YcDbKeenwnTSedUGng9KBAGJEDmY2MwfTnPvSbsfC7QSDHQkmB8vzpGALMFJIaURto+yuppzEG2gJfKB/iInpp568qsHIOFYbIx56GD/NtHKreqrhOIDEjmACdZOvn7GrWglyWjP9qeDfrFuUA7xNV5SOaE+f1965ris9skXLbpH3lIHzIg+1dqpJWsN+hrte7hm/Ta6VC24yjhh4m1tkv2SO8tksvMEEZXUgbgg7eVU/GONXcU4uX7mYgQuwVROyqNB9TXfsTwjD3P3lm238SKfyprB8Awlps9rD2kf7620D/5omtug/4sWsZ8Gb1CUNU1JZT7+DjPBOx+NxBBW0baH/tLvgEdQh8bewjzrtfBeHLh7FuymyKFnYk7sxHUkk+9T6FOtvlZzwZK6Y18B0KFCkjgUKFCoQrlqPj/AN2/8L/ShQp0+GYK/voruB/Bg/8Ag2/+mKifpQ/2P+YUKFJXB0PxEbtF/utP5fpWI7QfubHp+Ro6FR8Fx5KnCb+1dab/AHQv/wBIn/SWhQoY8MlnKOYfo/8A9tT/AL3/APENazAfH/fShQoq+CrfvGxw/wCVKw3xP7fU0KFOFk+1t7n6mqzim6fxP9aFCoUjjP6Tv95X/wCCz/0LdZdv3Te/0NChSw2eheD/ALm1/wAK3/yJQufEv8R/6bUKFEuAVyWfAfte30WrqhQoJchhUdChVECoUKFQgdFR0KhAUKFCoQFChQqEP//Z" class="image3">
```
Comment (jugalkpatel) [inline index.html:190]: try to use svgs in imgs

File: index.html [hunk 8]
```diff
@@ -0,0 +1,676 @@
...
+
+                <h3>Code for button component</h3>
+
+                <iframe src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=css&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cdiv%2520class%253D%2522button_head%2522%253E%250A%253Cbutton%2520id%253D%2522btn-default%2522%253EDefault%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn-primary%2522%253EPrimary%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn-secondary%2522%253ESecondary%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn-info%2522%253EInfo%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn-sucess%2522%253ESucess%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn-error%2522%253EError%253C%252Fbutton%253E%250A%253C%252Fdiv%253E%250A"
+                    style="width: 534px; height: 348px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+              </iframe>
+
+            </div>
+
+            <!-- IMAGE COMPONENT  -->
```
Comment (jugalkpatel) [inline index.html:238]: img is not responsive

File: index.html [hunk 9]
```diff
@@ -0,0 +1,676 @@
...
+                <br>
+                <h2>a. Responsive Image</h2>
+                <img class="responsive_img" src="https://www.w3schools.com/howto/img_nature_wide.jpg" alt="" srcset="">
+                <br/>
+                <h3>Code for responsive image:</h3>
+                <iframe src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=css&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520max-width%253A%2520100%2525%253B%250A%2520%2520%2520%2520display%253A%2520block%253B%250A%2520%2520%2520%2520height%253A%2520auto%253B%250A%2520%2520%2520%2520"
+                    style="width: 310px; height: 300px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+</iframe>
+                <br><br>
+                <h2>b. Round Image</h2>
```
Comment (jugalkpatel) [inline index.html:252]: round image is not responsive

File: index.html [hunk 10]
```diff
@@ -0,0 +1,676 @@
...
+                <h3>Code for round image:</h3>
+                <iframe src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=css&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=.round_img%2520%257B%250A%2520%2520%2520%2520border-radius%253A%252050%2525%253B%250A%2520%2520%2520%2520width%253A%2520400px%253B%250A%2520%2520%2520%2520height%253A%2520400px%253B%250A%2520%2520%2520%2520%250A%257D"
+                    style="width: 334px; height: 350px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+            </div>
+
+            <br>
+
+            <!-- CARD COMPONENT -->
```
Comment (jugalkpatel) [inline index.html:268]: add space between title and border and change font-size of the card title.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,676 @@
...
+
+                </div>
+
+                <iframe src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=css&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid_area1%2522%253E%250A%250A%253Cdiv%2520class%253D%2522grid1%2522%253EGrid%25201%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid2%2522%253EGrid%25202%253C%252Fdiv%253E%250A%250A%253C%252Fdiv%253E"
+                    style="width: 498px; height: 294px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+                <h2>Simple grid with three items</h2>
+
+                <div class="grid_area2">
```
Comment (jugalkpatel) [inline index.html:521]: Suggestion: Vertical item Grid is more useful.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,676 @@
...
+            <br><br><br>
+            <br><br><br>
+            <br><br><br>
+
+
+
+
+            <!-- MODAL COMPONENT -->
+
+            <div class="modal_header" id="modal_header">
```
Comment (jugalkpatel) [inline index.html:580]: modal cancel button is overflowing container.
image

File: index.html [hunk 13]
```diff
@@ -0,0 +1,676 @@
...
+
+
+
+        </div>
+
+    </main>
+
+</body>
+
+</html>
```
Comment (jugalkpatel) [inline index.html:676]: try to avoid br tags.

File: index.html (hunk 12 above)
Comment (Ayush-kumar1) [reply index.html:580]: When is it overflowing?? I can't find this issue??

File: index.html (hunk 7 above)
Comment (Ayush-kumar1) [reply index.html:190]: I am copying the address of an image from the internet, it's not a downloaded image. How can I change the format??

File: home.html [hunk 14]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
```
Comment (thesudeshdas) [inline home.html:1]: Add a favicon. Maybe that little ninja

File: home.html [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
```
Comment (thesudeshdas) [inline home.html:8]: Change the title to "UI Ninja | Home"

File: index.html [hunk 16]
```diff
@@ -0,0 +1,676 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Component Project</title>
```
Comment (thesudeshdas) [inline index.html:8]: Change the title to "UI Ninja | Docs". This sounds much better

File: index.html [hunk 17]
```diff
@@ -0,0 +1,676 @@
+<!DOCTYPE html>
```
Comment (thesudeshdas) [inline index.html:1]: Add mobile responsiveness in the next version

File: index.html [hunk 18]
```diff
@@ -0,0 +1,676 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Component Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css">
+</head>
+
+<body>
+
```
Comment (thesudeshdas) [inline index.html:16]: Why so much unneccesary space? Use some code formatter ([Prettier](https://prettier.io/))

File: index.html (hunk 3 above)
Comment (thesudeshdas) [reply index.html:11]: +1

File: index.html [hunk 19]
```diff
@@ -0,0 +1,676 @@
...
+
+    <main>
```
Comment (thesudeshdas) [inline index.html:18]: There is an horizontal scroll on the page, looks odd. Its due to one or more element going out of viewport. Look into that please

File: index.html [hunk 20]
```diff
@@ -0,0 +1,676 @@
...
+
+        <div id="page-wrap">
+            <div class="blok" id="sidebar">
+
+                <!-- <br><br> -->
+
+                <h3>Components</h3>
+
+                <ul>
+                    <!-- <li><a href="#header">Header</a></li> -->
```
Comment (thesudeshdas) [inline index.html:52]: Why does clicking on sidebar links take me to another another page, shouldn't they scroll to the section in the doc page itself?

File: index.html [hunk 21]
```diff
@@ -0,0 +1,676 @@
...
+            <div>
+                <h1>Introduction</h1>
+                <br>
+                <p>A lightweight and modular front-end framework for developing fast and powerful web interfaces.</p>
+
+
+            </div>
+            <br>
+
+            <!-- AVATAR COMPONENT -->
```
Comment (thesudeshdas) [inline index.html:82]: What is the difference between the components in the docs page & the component html page?

File: index.html [hunk 22]
```diff
@@ -0,0 +1,676 @@
...
+                    <li><a href="Grid.html">Grid</a></li>
+                    <li><a href="Rating.html">Rating</a></li>
+                    <li><a href="Modal.html">Modal</a></li>
+                    <li><a href="Navigation.html">Navigation</a></li>
+                    <li><a href="Dropdown.html">Dropdown</a></li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="blok" id="content">
```
Comment (thesudeshdas) [inline index.html:71]: Give some padding on the left & right of the components. It looks odd.

File: index.html [hunk 23]
```diff
@@ -0,0 +1,676 @@
...
+                <img src="avatar.svg" class="image">
+            </div>
+
+            <br>
+            <br>
+
+            <h3>Code for the avatar component:-</h3>
+
+
+            <iframe src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=css&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520id%253D%2522avatar_heading%2522%253E%250A%2520%253Cimg%2520src%253D%2522avatar.svg%2522%2520class%253D%2522image3%2522%253E%250A%2520%253Cimg%2520src%253D%2522avatar.svg%2522%2520class%253D%2522image2%2522%253E%250A%2520%253Cimg%2520src%253D%2522avatar.svg%2522%2520class%253D%2522image%2522%253E%250A%253C%252Fdiv%253E"
```
Comment (thesudeshdas) [inline index.html:98]: Some code snippets have vertical scroll, it is not needed IMO

File: index.html [hunk 24]
```diff
@@ -0,0 +1,676 @@
...
+
+            <span class="avatar_badge">✔️</span>
+
+                 </button>
+                </div>
+                <br>
+                <br>
+                <h3>Code for badge component</h3>
+
+                <iframe src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=css&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cdiv%2520class%253D%2522display%2522%253E%250A%2520%253Cbutton%2520type%253D%2522button%2522%2520class%253D%2522badge_button%2522%253E%250A%2520%253Cspan%2520class%253D%2522material-icons_custom%2522%253E%250A%2520notifications%250A%2520%253C%252Fspan%253E%250A%250A%2520%253Cspan%2520class%253D%2522icon_badge%2522%253E2%253C%252Fspan%253E%250A%250A%2520%253C%252Fbutton%253E%250A%2520%253C%252Fdiv%253E"
```
Comment (thesudeshdas) [inline index.html:200]: Everything needs to be aligned with the same basis

File: index.html [hunk 25]
```diff
@@ -0,0 +1,676 @@
...
+
+            <div id="rating_header">
+
+
+
+                <div class="rating-css">
+
+                    <h1>11. Rating Component</h1>
+                    <div class="star-icon">
+                        <input type="radio" name="rating1" id="rating1">
```
Comment (thesudeshdas) [inline index.html:547]: The last star is getting into next line

**Review by thesudeshdas — CHANGES_REQUESTED** [review]
Make changes to the folder structure, put the css files in one file. Write a README file for the project.

**Pull request merged**